biogeochem/EDPhysiologyMod.F90

```diff
@@ -366,6 +410,12 @@ subroutine phenology( currentSite, bc_in )
     integer  :: day                      ! day of month (1, ..., 31)
     integer  :: sec                      ! seconds of the day
 
+    real(r8) :: leaf_c                       ! leaf carbon [kg]
```

Comment:
minor, comment alignment is off (my OCD is showing)

Reply:
You know you can suggest edits with the new button for these things?

Reply:
I couldnt figure out how to "align" it with the other comments in that block otherwise I would have done that ;)